### ISSUE TITLE
Suggestion for the custom CSS support

### ISSUE THREAD

**Issue by voronind-com** [nkanaev/yarr#288]
Hello there. I really appreciate what you did with this reader. Boring tech is the best tech.

However, accessibility is a good thing. Black background is cool and all, but hard on eyes. Fonts and icons are also too small on a phone. The font only adjustable in the reader etc. I understand that your app is feature-complete, so this is just a mere suggestion that potentially covers a myriad of other requests. Currently I patch the [font & icon sizes](https://git.voronind.com/voronind/nix/src/commit/8b93059a4604a117a686944bf1b909835d4ccc24/package/yarr/font-size.patch) and [colors](https://git.voronind.com/voronind/nix/src/commit/8b93059a4604a117a686944bf1b909835d4ccc24/package/yarr/color.patch) for myself and don't really need this. So if you disagree with my suggestion, feel free to close it.

I'll leave some screenshots so that you could see the difference.
Thanks!

<img width="200" alt="Image" src="https://github.com/user-attachments/assets/8c44d67f-3261-4a11-814c-6ec2b2aa92c3" />

<img height="200" alt="Image" src="https://github.com/user-attachments/assets/d2c6ddcc-250d-4fe3-82f9-0999fe1b4c7b" />
<img height="200" alt="Image" src="https://github.com/user-attachments/assets/7b4c59a3-b707-4daf-8f7b-2087a201b029" />
<img height="200" alt="Image" src="https://github.com/user-attachments/assets/e082f0b8-43c4-421a-a107-23ba7507b8a1" />

**Comment by nkanaev** [comment]
Thanks for the feedback. I really like your customisation.

There's a lot of room to improve accessibility. However, the changes you've suggested mostly seem to fall into visual aesthetics rather than improving the usability for visually impaired users.

Aesthetics is subjective, but changes in css to make ricing easier (via vars) is always welcome.
